test/e2e/knative_serving_test.go

```diff
@@ -73,9 +73,12 @@ func TestKnativeServing(t *testing.T) {
 		testKnativeVersusKubeServicesInOneNamespace(t, caCtx)
 	})
 
-	t.Run("update global proxy and verify calls goes through proxy server", func(t *testing.T) {
-		testKnativeServingForGlobalProxy(t, caCtx)
-	})
+	// t.Run for proxy testing
+	t.Skip("update global proxy and verify calls goes through proxy server")/*
+		func(t *testing.T) {
```

Comment:
You shouldn't need to comment this out. The Skip should do the right thin, I think? Same goes for the `t.Run` call.

Reply:
`func (c *common) Skip(args ...interface{}) {}` just take args not func

Reply:
Right, but you can skip an intact test just fine. In this case

```golang
t.Run("update global proxy and verify calls goes through proxy server", func(t *testing.T) {
  t.Skip("Skipped because the test takes too long to run for now")
  testKnativeServingForGlobalProxy(t, caCtx)
}
```

Should work just fine.